packages/trace-viewer/src/ui/attachmentsTab.tsx

```diff
@@ -56,8 +63,9 @@ const ExpandableAttachment: React.FunctionComponent<ExpandableAttachmentProps> =
     return Math.min(Math.max(5, lineCount), 20) * lineHeight;
   }, [attachmentText]);
 
-  const title = <span style={{ marginLeft: 5 }}>
-    {linkifyText(attachment.name)} {hasContent && <a style={{ marginLeft: 5 }} href={downloadURL(attachment)}>download</a>}
+  const title = <span style={{ marginLeft: 5 }} ref={ref} title={attachment.name}>
+    <span style={highlight ? { textDecoration: 'underline var(--vscode-terminal-findMatchBackground)', textDecorationThickness: 1.5 } : {}}>{linkifyText(attachment.name)}</span>
```

Comment:
We usually add a className conditionally, and then put all styles into a css file.